actionview/lib/action_view/helpers/csp_helper.rb

```diff
@@ -14,9 +14,11 @@ module CspHelper
       # This is used by the Rails UJS helper to create dynamically
       # loaded inline <script> elements.
       #
-      def csp_meta_tag
+      def csp_meta_tag(options = {})
```

Comment:
How about using splat hash?

Since Ruby 2.6, this options hash would cause a warning like here:

```ruby
def foo(a = {}) 
  a
end

a = { foo: "foo" }

p foo(a.merge(bar: "bar"))
p foo(**a, bar: "bar")
```

```
% ruby -w xxx.rb
{:foo=>"foo", :bar=>"bar"}
xxx.rb:1: warning: in `foo': the last argument was passed as a single Hash
xxx.rb:8: warning: although a splat keyword arguments here
{:foo=>"foo", :bar=>"bar"}
```

Reply:
Thanks, make sense. I fixed.